integration/env/env.go

```diff
@@ -71,13 +78,30 @@ func CircleSHA() string {
 }
 
 func ClusterID() string {
-	return fmt.Sprintf("ci-awsop-%s-%s", TestedVersion(), CircleSHA()[0:5])
+	return fmt.Sprintf("ci-awsop-%s-%s", TestedVersion(), TestHash())
```

Comment:
One odd thing is that we would not have the Git SHA visible. I fear the IDs are getting too long when we add more information to it when we would just append another hash for the test dir. The IDs being too long is another retarded problem in itself but we cannot just workaround this right now for several reasons. One option here would be to drop the `ci-` or `aws-op` part to have the Git SHA and the test dir hash separated. WDYT?